src/core_plugins/console/api_server/es_6_0/aggregations.js

```diff
@@ -0,0 +1,514 @@
+/*eslint camelcase: 0*/
```

Comment:
What's the value in disabling the camelcase rule for one file? Why not just use camelCase?

Reply:
the API uses camel case.

Reply:
This override is just for local variable names which don't need to be snake_case, even if the API does expect snake_case.